Makefile

```diff
@@ -1,12 +1,12 @@
-# Add build flags to GOFLAGS=<flags> here.
 GO := go
+GO_BUILD_FLAGS := -tags=json1
```

Comment:
Some of these changes are from a prior commit in main. Can you rebase so they go away?